docs/services/frame-system/_index.md

```diff
@@ -91,18 +94,104 @@ You can use [the right-hand rule](https://en.wikipedia.org/wiki/Right-hand_rule)
 
 For more information about determining the appropriate values for these parameters, see these two examples:
 
-- [A component attached to a static surface](/services/frame-system/component-on-static)
-- [A component attached to another, dynamic, component](/services/frame-system/component-on-dynamic)
+- [A Reference Frame:](/services/frame-system/frame-config) A component attached to a static surface
+- [Nested Reference Frames:](/services/frame-system/nested-frame-config) A component attached to another, dynamic, component
+
+### Visualize the Frame System
+
+You can visualize how your robot is oriented in the Frame System in [the Viam app](https://app.viam.com).
+Navigate to the **config** tab on your robot's page, select **mode** as **Builder**, and click on **Frame System**.
+
+The Viam app shows you a 3D visualization of the spatial configuration of the reference frames of components configured on your robot:
+
+![Default frame system configuration grid visualization for a single component, shown in the Frame System Editor](img/frame_system_basic.png)
+
+This tab provides a simple interface for simultaneously viewing and editing the position, orientation, and geometries of a robot's components in the Frame System.
+
+For example:
+
+Consider a robot configured with a [`jetson` board](/components/board), wired to a [`webcam` camera](/components/camera/webcam) and a [`wheeled` base](/components/base/wheeled) with two [motors](/components/motor) driving its wheels.
+
+You have not yet added any reference frame configuration to these components.
+If you go to the **config** tab of your robot and click on the **Frame System** sub-tab, they are all shown to be located on the default `world` origin point as follows:
+
+  ![Example robot's default frame configuration shown in the Frame System Editor. All components are stuck on top of each other](img/demo_base_unedited.png)
+
+You measure the distance on the floor from your base to your board and camera setup and find it to be 200 millimeters.
+If you add this value to `"X"` in your base's reference frame `Translation` attribute, the Frame System readjusts to show the base's translation:
+
+  ![Base translated 200mm forwards shown in the Frame System Editor](img/demo_base_edited.png)
+
+Next, you measure the distance from your board to the camera mounted overhead and find it to be 50 millimeters.
+If you add this value to `"Z"` in your camera's reference frame `Translation` attribute, the Frame System readjusts to show the camera's translation:
```

Comment:
Since this is all an example I find the `you` strange in all of this.

Reply:
Yeah it's a little weird.

 I don't know how to translate directions like `You have not yet added any reference frame configuration to these components.` without speaking to the user or using *we*